HW1.py

```diff
@@ -0,0 +1,120 @@
+import sys
+
+import pandas as pd
+import plotly.express as px
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.tree import DecisionTreeClassifier
+
+
+def main():
+
+    # Load Data
+    iris_data = pd.read_csv("iris.data")
```

Comment:
The first observation is being imported as a header, you may want to fix this.

Reply:
Fixed this, added header=None

Reference: ae50f75027bf6f9296028511081cfe43ebea0912

Reply:
Nice catch!